smslibrary/build.gradle

```diff
@@ -37,5 +37,10 @@ dependencies {
     testImplementation 'org.mockito:mockito-core:1.10.19'
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
-    implementation 'com.github.CremaLuca:android-preferences:2.3'
+
+    androidTestImplementation 'com.android.support.test:rules:1.0.2'
+    androidTestImplementation 'org.mockito:mockito-android:3.2.4'
```

Comment:
These two dependencies are not needed, as you can see they're for instrumented tests but we don't have any of those. I had added them temporarily in #20 when I was writing instrumented test, but I couldn't get them to compile so I removed them in the end.